<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomeopathyDrugForm</class>
 <widget class="QWidget" name="HomeopathyDrugForm">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>296</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="errorLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #ff0000;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="nameLabel">
       <property name="minimumSize">
        <size>
         <width>135</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>135</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Наименование</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="name">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="nameLatLabel">
       <property name="minimumSize">
        <size>
         <width>135</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>135</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Наименование (лат.)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nameLat">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="groupLabel">
       <property name="minimumSize">
        <size>
         <width>135</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>135</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Группа</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="groupComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="dilutionsLabel">
       <property name="minimumSize">
        <size>
         <width>135</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>135</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="locale">
        <locale language="Russian" country="Russia"/>
       </property>
       <property name="text">
        <string>Разведения</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="dilutions">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addDilutionsBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Добавить</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/icons/greenPlusIcon.png</normaloff>:/icons/greenPlusIcon.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
